### PR TITLE
[pt] Add Expose Your App Publicly in Portuguese

### PR DESCRIPTION
**Pull request by AlexDamiao86**
Add Expose Your App Publicly Directory in Portuguese 
Update Index to link to the translated page.

### CODE REVIEW

File: content/pt/docs/tutorials/kubernetes-basics/expose/expose-interactive.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+<link href="/docs/tutorials/kubernetes-basics/public/css/overrides.css" rel="stylesheet">
+<script src="https://katacoda.com/embed.js"></script>
+
+<div class="layout" id="top">
+
+    <main class="content katacoda-content">
+
+        <div class="katacoda">
+            <div class="katacoda__alert">
+                Para interagir com seu terminal, favor utilizar a versão desktop/tablet
```
Comment (yagonobre) [inline content/pt/docs/tutorials/kubernetes-basics/expose/expose-interactive.html:22]: "Para interagir com o terminal"

File: content/pt/docs/tutorials/kubernetes-basics/expose/expose-interactive.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            </div>
+            <div class="katacoda__box" id="inline-terminal-1"  data-katacoda-id="kubernetes-bootcamp/8" data-katacoda-color="326de6" data-katacoda-secondary="273d6d" data-katacoda-hideintro="false" data-katacoda-prompt="Terminal Kubernetes Bootcamp" style="height: 600px;">
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md-12">
+                <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/scale/scale-intro/" role="button">Continue com o Módulo 5<span class="btn__next">›</span></a>
```
Comment (yagonobre) [inline content/pt/docs/tutorials/kubernetes-basics/expose/expose-interactive.html:29]: Continuar para o

File: content/pt/docs/tutorials/kubernetes-basics/expose/expose-intro.html [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+			<p>Um serviço no Kubernetes é uma abstração que define um conjunto lógico de Pods e uma política pela qual acessá-los. Serviços permitem um baixo acoplamento entre os Pods dependentes. Um serviço é definido usando YAML <a href="/docs/concepts/configuration/overview/#general-configuration-tips">(preferencialmente)</a> ou JSON, como todos objetos Kubernetes. O conjunto de Pods selecionados por um Serviço é geralmente determinado por um seletor de rótulos <i>LabelSelector</i> (veja abaixo o motivo pelo qual você pode querer um Serviço sem incluir um seletor <code>selector</code> na especificação <code>spec</code>).</p>
+
+			<p>Embora cada Pod tenha um endereço IP único, estes IPs não são expostos externamente ao cluster sem um Serviço. Serviços permitem que suas aplicações recebam tráfego. Serviços podem ser expostos de formas diferentes especificando um tipo <code>type</code> na especificação do serviço <code>ServiceSpec</code>:</p>
+			<ul>
+				<li><i>ClusterIP</i> (padrão) - Expõe o serviço sob um endereço IP interno no cluster. Este tipo faz do serviço somente alcancável a partir de dentro do cluster.</li>
```
Comment (yagonobre) [inline content/pt/docs/tutorials/kubernetes-basics/expose/expose-intro.html:37]: alcancável de dentro do cluster.

**Review by yagonobre — COMMENTED** [review]
Thanks for your contribution, I added few comments from a first pass.

File: content/pt/docs/tutorials/kubernetes-basics/expose/expose-interactive.html (hunk 2 above)
Comment (AlexDamiao86) [reply content/pt/docs/tutorials/kubernetes-basics/expose/expose-interactive.html]: Done!

File: content/pt/docs/tutorials/kubernetes-basics/expose/expose-intro.html (hunk 3 above)
Comment (AlexDamiao86) [reply content/pt/docs/tutorials/kubernetes-basics/expose/expose-intro.html]: Done!

**Comment by yagonobre** [comment]
Thanks!
/approve
/lgtm

**Pull request merged**